web/src/main/java/co/mobileaction/example/web/controller/PostController.java

```diff
@@ -41,4 +42,26 @@ public ResponseEntity<Boolean> deletePost(@PathVariable Long postId)
 
         return ResponseEntity.ok(true);
     }
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping
+    public void createPost(@RequestBody Post post)

```

Comment:
generally we don't directly get body as database entity class, it will be better to take it as a dto (say PostDto), and convert it to the entity. In that way, entity and body will not be coupled.